### PR TITLE
Necessary changes for on-prem deploys again

### PR DESCRIPTION
**Pull request by JoshOldenburg**
Removes some of the changes made to support Vapor and reverts to requiring PHP 7.2.

### CODE REVIEW

File: app/Jobs/CreateOrUpdateUserFromBuzzAPI.php [hunk 1]
```diff
@@ -64,7 +70,7 @@ public function __construct(string $identifier, $value, string $reason)
         $this->identifier = $identifier;
         $this->value = $value;
         $this->reason = $reason;
-        $this->queue = config('vapor-queue-names.buzzapi');
+        $this->queue = config('queue.name.buzzapi');
```
Comment (kberzinch) [inline app/Jobs/CreateOrUpdateUserFromBuzzAPI.php:73]: These should go back to being constants

File: config/queue.php [hunk 2]
```diff
@@ -88,4 +88,10 @@
         'table' => 'failed_jobs',
     ],
 
+    'name' => [
+        'email' => (env('QUEUE_CONNECTION') === 'redis' ? '' : (env('CACHE_PREFIX').'-')).'email',
+        'slack' => (env('QUEUE_CONNECTION') === 'redis' ? '' : (env('CACHE_PREFIX').'-')).'slack',
+        'buzzapi' => (env('QUEUE_CONNECTION') === 'redis' ? '' : (env('CACHE_PREFIX').'-')).'buzzapi',
+        'jedi' => (env('QUEUE_CONNECTION') === 'redis' ? '' : (env('CACHE_PREFIX').'-')).'jedi',
+    ],
```
Comment (kberzinch) [inline config/queue.php:96]: Delete this

File: phpcs.xml [hunk 3]
```diff
@@ -221,9 +221,7 @@
         <exclude-pattern>/app/Http/Middleware/</exclude-pattern>
     </rule>
 
-    <rule ref="SlevomatCodingStandard.TypeHints">
-        <exclude name="SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingNativeTypeHint"/>
```
Comment (kberzinch) [inline phpcs.xml:225]: You do want to keep this. Native property type hints are 7.4 only.

**Review by kberzinch — CHANGES_REQUESTED** [review]
Please review the colo milestone and attach the appropriate tickets to this PR: https://github.com/RoboJackets/apiary/milestone/13

**Comment by JoshOldenburg** [comment]
Fixes #1339, #1341, and #1342.

**Comment by JoshOldenburg** [comment]
This is a diff from before the Vapor PR to this PR: https://github.com/RoboJackets/apiary/compare/fb8214...josh/onprem

I read through that diff and I don't see anything problematic.

**Review by kberzinch — APPROVED** [review]

**Pull request merged**